orttraining/orttraining/python/training/optim/fused_adam.py

```diff
@@ -12,13 +12,22 @@
 
 import torch
 from ._multi_tensor_apply import MultiTensorApply
+from enum import IntEnum
```

Comment:
AdamWMode is not using it?

Reply:
that makes sense. I'll make the c++ enum accessible to python.

Reply:
Thinking about this a little more, doing this might not be so trivial.
The problem is that to expose the c++ enum to users, we need to define it in the `__init__` so users can do something like `from onnxruntime.training.optim import AdamWMode`. But there are users who don't need `FusedAdam` or `AdamWMode`. And these users may not have the torch cpp extensions installed. Exposing this common enum to all users would mean that we need to force users to install the torch cpp extensions (even if they don't want to use the `FusedAdam` or `AdamWMode`). 
Please let me know if this does not make sense, and we can discuss this further.